flink-core/src/main/java/org/apache/flink/core/plugin/PluginLoader.java

```diff
@@ -182,5 +184,16 @@ private boolean isAllowedFlinkClass(final String name) {
 		private boolean isAllowedFlinkResource(final String name) {
 			return Arrays.stream(allowedResourcePrefixes).anyMatch(name::startsWith);
 		}
+
+		static {
+			ClassLoader platformLoader = null;
+			try {
+				platformLoader = (ClassLoader) ClassLoader.class
+					.getMethod("getPlatformClassLoader")
+					.invoke(null);
+			} catch (Exception e) {
+			}
```

Comment:
Shouldn't we propagate/handle some errors?

Reply:
This will always fail on Java 8, so not sure what to handle here. I also looked into detecting the Java version and do only specific stuff, but everything is more fragile than just trying.

Reply:
We could make it more differentiated and just swallow `NoSuchMethodException` though.